common/pop_jobs/hfe_job.txt

```diff
@@ -1132,20 +1220,726 @@ hfe_job_recycler_hive_mind = {
+		is_enslaved = no
+	}
+
+	possible = {
+		owner = { has_country_flag = hfe_varelse_primitive_0 }
+		owner = {
+			NOT = {
+				has_country_flag = hfe_varelse_fedora_governance
+				has_civic = hfe_civic_overthrown_mechanicus
+				is_materialist = yes
+			}
+		}
+		OR = {
+			has_citizenship_rights = yes
+			has_citizenship_type = {
+				type = citizenship_full
+			}
+			owner = { is_country_type = primitive }
+			owner = { is_primitive = yes }
+			is_same_species = owner
+		}
+	}
+
+	resources = {
+		category = hfe_godforge
+		produces = {
+			alloys = 1
+			unity = 15
+		}
+		upkeep = {
+			trigger = {
+				OR = {
+					owner = { NOT = { has_country_flag = hfe_varelse_primitive_0 } }
+					owner = {
+						NOT = {
+							is_country_type = primitive
+							is_primitive = yes
+						}
+					}
+				}
+			}
+			food = 3
+			energy = 2
+		}
+	}
+
+#	country_modifier = {
+#		category_military_theory_research_speed_mult = 0.25
+#		category_materials_research_speed_mult = 0.1
+#		# As opposed to the High Mechanicus himself being a Job
+#		# Unfortunately, the game does not calculate Armies
+#		diplo_weight_pops_mult = 0.05
+#	}
+
+	planet_modifier = {
+#		army_collateral_damage_mult = 0.2
+#		army_health = 0.03
+#		army_morale = 0.25
+		planet_crime_add = -25
+		pop_defense_armies_add = 1
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = {
+				has_country_flag = hfe_varelse_primitive_0
+				NOT = {
+					is_primitive = yes
+					is_country_type = primitive
+				}
+			}
+		}
+		pop_defense_armies_add = 1
+	}
+
+	pop_modifier = {
+		pop_political_power = 0.5
+	}
+
+	weight = {
+		weight = 5
+		modifier = {
+			factor = 0.01
+			AND = {
+				NOT = {	is_same_species = owner }
+				owner = {
+					NOT = {
+						is_primitive = yes
+						is_country_type = primitive
+					}
+				}
+				owner = {
+					NOT = {
+						is_xenophile = yes
+						is_egalitarian = yes
+					}
+				}
+			}
+		}
+		modifier = {
+			factor = 20
+			is_exact_same_species = owner
+		}
+		modifier = {
+			factor = 10
+			is_same_species = owner
+		}
+		# crisis purge
+		modifier = {
+			factor = 0
+			planet.controller = {
+				OR = {
+					is_country_type = swarm
+					is_country_type = ai_empire
+				}
+			}
+		}
+	}
+}
+
+hfe_job_mechanicus_cultist = {
+	category = worker
+	building_icon = building_underground_chemists
+	icon = organic_battery
+	can_set_priority = no
+	clothes_texture_index = 6
+	is_capped_by_modifier = yes
+
+	possible_pre_triggers = {
+		has_owner = yes
+		is_enslaved = no
+		is_being_purged = no
+		is_being_assimilated = no
+	}
+
+	possible = {
+		OR = {
+			planet = { has_building = hfe_building_mechanicus_cult_temple }
+			planet = { has_planet_flag = hfe_colony_beta }
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+			owner = { is_primitive = yes }
+			owner = { is_country_type = primitive }
+		}
+		OR = {
+			has_citizenship_type = {
+				type = citizenship_full
+			}
+			is_same_species = owner_species
```

Comment:
is it possible to main species do not have sitizen rights?

Reply:
No. Why does that matter?